tests/test_lora.py

```diff
@@ -384,37 +384,34 @@ def test_lora_merge_with_quantize():
         model = GPT(config)
         model.apply(model._init_weights)
 
+    mark_only_lora_as_trainable(model)
+
     optimizer = bnb.bnb.optim.PagedAdamW(model.parameters(), lr=1.0)
```

Comment:
is it necessary to import bnb through our module?

Reply:
No, changed it